src/config/bookconfig.rs

```diff
@@ -17,6 +18,8 @@ pub struct BookConfig {
     multilingual: bool,
     indent_spaces: i32,
 
+    playpen_config: PlaypenConfig,
```

Comment:
`HtmlConfig` doesn't seem to have correspondence to anything in the code so I wasn't sure what it meant. Maybe the folder you're calling `theme` should truly be `html_renderer` or something then, which would then explain why you want me to move `editor` there?